histomicsui/web_client/stylesheets/panels/zoomWidget.styl

```diff
@@ -2,13 +2,15 @@
   .h-zoom-slider
     padding 0
     box-shadow none
+    width 70%
```

Comment:
I think I'd rather see this as `calc(100% - 76px);`, since that way it will fill the available space regardless of the width of the scroll bar or lack thereof.